src/herbie.rkt

```diff
@@ -12,6 +13,18 @@
 (define (string->thread-count th)
   (match th ["no" #f] ["yes" (max (- (processor-count) 1) 1)] [_ (string->number th)]))
 
+(define (operator-pruning)
+  (prune-operators!)
+  (prune-rules!)
+  (unless (if (flag-set? 'precision 'double) (null? (*unknown-d-ops*)) (null? (*unknown-f-ops*)))
```

Comment:
I think it'd be clearer to write this by hoisting the `null?` out of the conditional:

    (null? (if (flag-set? 'precision 'double) (*unknown-d-ops*) (*unknown-f-ops*)))